api/src/pcapi/core/token/__init__.py

```diff
@@ -25,7 +25,7 @@ class Token:
     data: dict
 
     @classmethod
-    def load(cls: "Token", encoded_token: str) -> "Token":
+    def load(cls, encoded_token: str) -> "Token":
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏Aah c'est fait ici
Tu peux mettre ça dans le commit avant plutôt

Ça se fait bien c'est tout le fichier qui est concerné :wink: